tests/tape/test_qscript.py

```diff
@@ -231,7 +231,7 @@ def test_get_operation(self):
         assert op_6 == ops[4] and op_id_6 == 4 and p_id_6 == 1
 
         _, obs_id_0, p_id_0 = qs.get_operation(7)
-        assert obs_id_0 == 0 and p_id_0 == 0
+        assert obs_id_0 == 5 and p_id_0 == 0
```

Comment:
Just leaving a note here in case this change is reverted.

Reply:
Reverted.